### PR TITLE
ENG-2519 basic content attributes scenario tests

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: ui-tests/cypress/support/command/command.js [hunk 1]
```diff
@@ -28,4 +28,26 @@ Cypress.Commands.add('uploadRequest', ({ method, url, body, headers, auth }) =>
   });
 });
 
+Cypress.Commands.add('uploadRequest', ({ method, url, body, headers, auth }) => {
```
Comment (g-di-martino) [inline ui-tests/cypress/support/command/command.js:31]: isn't it the same as line 8?

File: ui-tests/cypress/support/restAPI/assetsAPI.js [hunk 2]
```diff
@@ -33,6 +33,16 @@ class AssetsController {
       )
       .then(response => ({ controller: this, response: JSON.parse(response) }));
   }
+  
+  getAssetsList(type = 'image') {
+    return cy.request({
+      url: `${controller}?type=${type}&sort=createdAt&direction=DESC&page=1&pageSize=10`,
```
Comment (g-di-martino) [inline ui-tests/cypress/support/restAPI/assetsAPI.js:39]: you can use _qs_ argument to define the query parameters (see [Cypress request](https://docs.cypress.io/api/commands/request#Arguments))

File: ui-tests/cypress/support/restAPI/contentsAPI.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ class ContentsController {
       body: [{
         ...content
       }]
-    });
+    }).then(response => ({ controller: this, response }));
```
Comment (g-di-martino) [inline ui-tests/cypress/support/restAPI/contentsAPI.js:36]: you can append the _then_ where you need it, in order to leave to each use case what to do with the result of the request

File: ui-tests/package.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
     "cypress": "^7.6.0",
     "cypress-file-upload": "^5.0.8",
     "cypress-keycloak-commands": "^1.2.0",
+    "cypress-real-events": "^1.5.1",
```
Comment (g-di-martino) [inline ui-tests/package.json:19]: what is this library for?

**Review by g-di-martino — CHANGES_REQUESTED** [review]
Maybe it's worth to document the changes and have a session to explain it to everybody; I can see you have built a quite complex structure

File: ui-tests/cypress/support/command/command.js (hunk 1 above)
Comment (jeffgo10) [reply ui-tests/cypress/support/command/command.js:31]: sorry. I must have slipped this out when I rebased the branch

File: ui-tests/cypress/support/restAPI/assetsAPI.js (hunk 2 above)
Comment (jeffgo10) [reply ui-tests/cypress/support/restAPI/assetsAPI.js:39]: done

File: ui-tests/cypress/support/restAPI/contentsAPI.js (hunk 3 above)
Comment (jeffgo10) [reply ui-tests/cypress/support/restAPI/contentsAPI.js]: make sense. I have made changes

File: ui-tests/package.json (hunk 4 above)
Comment (jeffgo10) [reply ui-tests/package.json:19]: this is for arrow key presses and text highlights for hypertext scenario test (`rich text editor` activated). the only text selection shortcuts for cypress was ever introduced was `{selectall}`. I wanted to do multiple text selections as what the test suggested so this is what most people suggested in that kind of scenario. See `Hypertext > try to add several internal link` scenario test for the actual usage.

**Comment by jeffgo10** [comment]
> Maybe it's worth to document the changes and have a session to explain it to everybody; I can see you have built a quite complex structure

regarding documentation, how do we proceed on it? Do we put it in readme? Yes I am free to discuss to teammates regarding the functions created

**Comment by g-di-martino** [comment]
> > Maybe it's worth to document the changes and have a session to explain it to everybody; I can see you have built a quite complex structure
> 
> regarding documentation, how do we proceed on it? Do we put it in readme? Yes I am free to discuss to teammates regarding the functions created

We have a documentation page on the YouTrack [Knowledge base](https://entando.myjetbrains.com/youtrack/articles/ENG-A-9/Cypress-Automated-Testing), maybe you can add a sub-page there; then we can plan a discussion with teammates 😄

**Comment by jeffgo10** [comment]
> > > Maybe it's worth to document the changes and have a session to explain it to everybody; I can see you have built a quite complex structure
> > 
> > 
> > regarding documentation, how do we proceed on it? Do we put it in readme? Yes I am free to discuss to teammates regarding the functions created
> 
> We have a documentation page on the YouTrack [Knowledge base](https://entando.myjetbrains.com/youtrack/articles/ENG-A-9/Cypress-Automated-Testing), maybe you can add a sub-page there; then we can plan a discussion with teammates 😄

sounds like a plan. would it be okay if I put documentation in a different ticket?

File: ui-tests/package.json (hunk 4 above)
Comment (g-di-martino) [reply ui-tests/package.json:19]: 👍🏼

**Comment by g-di-martino** [comment]
> > > > Maybe it's worth to document the changes and have a session to explain it to everybody; I can see you have built a quite complex structure
> > > 
> > > 
> > > regarding documentation, how do we proceed on it? Do we put it in readme? Yes I am free to discuss to teammates regarding the functions created
> > 
> > 
> > We have a documentation page on the YouTrack [Knowledge base](https://entando.myjetbrains.com/youtrack/articles/ENG-A-9/Cypress-Automated-Testing), maybe you can add a sub-page there; then we can plan a discussion with teammates 😄
> 
> sounds like a plan. would it be okay if I put documentation in a different ticket?

I've created a new sub-task for the documentation ([ENG-2639](https://entando.myjetbrains.com/youtrack/issue/ENG-2639)), please ensure you complete that as well before closing the main task ([ENG-2519](https://entando.myjetbrains.com/youtrack/issue/ENG-2519))

**Review by g-di-martino — APPROVED** [review]

**Pull request merged**